fastapi_sqla/sqla.py

```diff
@@ -293,15 +255,15 @@ def _paginate(
     limit: int,
     *,
     scalars: bool = True,
-) -> Page[T]:
+) -> Page:
```

Comment:
It didn't made a lot of sense to return `[T]` here since the TypeVar wasn't bound to anything in that context. It would have made sense if for example we had the signature like:

```
def paginate_query(
    query: Select[T],
    session: Session,
    total_items: int,
    offset: int,
    limit: int,
    *,
    scalars: bool = True,
) -> Page[T]:
```